docs/examples/imageset-config-release-full-channel.yaml

```diff
@@ -7,4 +7,4 @@ mirror:
   ocp:
     channels:
       - name: stable-4.9
-        headsOnly: false
\ No newline at end of file
+        allVersions: true
```

Comment:
allversions combined with `shortestPath` doesnt make much sense. I suggest `all` here as well. That also helps keep parity between operators and ocp releases.

Reply:
Perhaps we should create a validation that disallows the combination between `shortestPath` and `allVersions` or `all`. I doubt needing the `shortestPath` between the min and max of a channel would be a very common use case. WDYT?